sdks/typescript/src/v1/declaration.ts

```diff
@@ -266,12 +266,52 @@ export class BaseWorkflowDeclaration<
     input: I,
     options?: RunOpts,
     _standaloneTaskName?: string
-  ): Promise<WorkflowRunRef<O>> {
+  ): Promise<WorkflowRunRef<O>>;
+  async runNoWait(
```

Comment:
this fn is now the source of truth for all the hacks we're doing -- everything else should roll up to this implementation